packages/autocertifier-client/src/createSelfSignedCertificate.ts

```diff
@@ -0,0 +1,78 @@
+import * as forge from 'node-forge'
+
+interface CertificateChain {
+    caCert: string
+    caKey: string
+    serverCert: string
+    serverKey: string
+}
+
+export function createSelfSignedCertificate(fqdn: string, validMonths: number): CertificateChain {
+    if (validMonths < 1) {
```

Comment:
To be sync with the error message

```suggestion
    if (validMonths <= 0) {
```